### PR TITLE
Change Monitoring API suffix from api to client

### PR DESCRIPTION
**Pull request by swcloud**

### CODE REVIEW

**Comment by quartzmo** [comment]
Looks like you might need to exclude `lib/google/cloud/monitoring/v3` from rubocop checks?

File: google-cloud-monitoring/.rubocop.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ AllCops:
     - "integration/**/*"
     - "google-cloud-monitoring.gemspec"
     - "lib/google/monitoring/**/*"
+    - "lib/google/cloud/monitoring/**/*"
```
Comment (quartzmo) [inline google-cloud-monitoring/.rubocop.yml]: Should we be concerned that this pattern will potentially exclude future hand-written code from rubocop checks?

Comment (swcloud) [inline google-cloud-monitoring/.rubocop.yml]: I would expect all the files under that directory should be auto-generated.

Comment (quartzmo) [inline google-cloud-monitoring/.rubocop.yml]: OK, sounds good.

**Comment by quartzmo** [comment]
Can you add a commit to update the names in [`google-cloud-monitoring/docs/toc.json`](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-monitoring/docs/toc.json#L15-L20)?

**Comment by swcloud** [comment]
@quartzmo Is it manually edited or auto-generated? If auto-generated, what's command to generate it?

**Comment by quartzmo** [comment]
`toc.json` is manually edited, since it is a curated list, rather than an index.

**Comment by quartzmo** [comment]
Oh shoot, I just found similar usages of the old names that need to be updated in google-cloud/docs/toc.json[](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud/docs/toc.json#L301-L306).

**Comment by quartzmo** [comment]
@swcloud I believe you working on similar name changes for Error Reporting and Pub/Sub? If so, can you add a commit to your next PR updating the Monitoring client names in [google-cloud/docs/toc.json](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud/docs/toc.json#L301-L306)?

**Comment by swcloud** [comment]
Will do.

**Pull request merged**